test/JSONResourceFilePaths.test.js

```diff
@@ -99,6 +99,27 @@ var p5 = new CustomProject({
     }
 });
 
+var p6 = new CustomProject({
+    id: "flutterHome",
+    projectType: "webos-dart",
+    sourceLocale: "en-KR",
+    resourceDirs: {
+        "json": "resources"
+    }
+    }, "./testfiles", {
+    localeMap: {
+        "es-CO": "es"
+    },
+    jsonMap: {
```

Comment:
Should be settings.dart.mappings instead of jsonMap.mappings. All of the plugin-specific settings go in the settings object, just like for all of the other plugins. eg. non-webos Javascript files have their translations in json files using the mappings in settings.javascript.mappings. You can see it used in https://github.com/iLib-js/ilib-loctool-javascript/blob/main/JavaScriptFileType.js#L81

Reply:
I updated the webos-dart pluugin. https://github.com/iLib-js/ilib-loctool-webos-dart/pull/5
The config file would like this
```
"settings" : {
   "dart": {
      "mappings" : {
         "**/*.dart": { "template": "[dir]/assets/i18n/[localeUnder].json"}
        }
     }
...
```